lib/user.rb

```diff
@@ -0,0 +1,24 @@
+class User
+  attr_reader :name, :jokes
+
+  def initialize(name, jokes = [])
+    @name = name
+    @jokes = jokes
+  end
+
+  def learn(joke)
+    @jokes << joke
+  end
+
+  def tell(user, joke)
+    user.jokes << joke
+  end
+
+  def joke_by_id(id)
+    # require "pry"; binding.pry
+    [@jokes.find do |joke|
```

Comment:
You should not need to wrap this in an array, we were just looking for the found joke